vanilla-framework/vanilla-framework
# Add Prettier ✨

## Done

- Add Prettier to the codebase
- Edit `.sass-lint` to remove indentation rule as Prettier takes care of that
- Add a `precommit` hook

## QA

- Pull code
- Run `./run serve --watch`
- Open http://0.0.0.0:8101/vanilla-framework/
- Ensure no tests fail
- Ensure no visual regressions
